vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Looks like some stuff got added in a merge commit that shouldn't be here. https://github.com/woothemes/woocommerce/pull/11540/commits/c61c08f405fc73a1b8ffa39765a1fdb0a9dc000f